core/roboconf-target-iaas-ec2/src/main/java/net/roboconf/target/ec2/internal/Ec2MachineConfigurator.java

```diff
@@ -192,23 +235,131 @@ private boolean checkVmIsStarted() {
 		dis.setInstanceIds(Collections.singletonList(this.machineId));
 
 		DescribeInstancesResult disresult = this.ec2Api.describeInstances( dis );
+		// Obtain availability zone (for later use, eg. volume attachment).
+		this.availabilityZone = disresult.getReservations().get(0).getInstances().get(0).getPlacement().getAvailabilityZone();
```

Comment:
Explain in a comment why we do not keep the availibility zone we got from the target properties.
In fact, just indicate it is optional and that we want to use the same than the VM if it was not specified.